src/decoder_wildmidi.cpp

```diff
@@ -175,13 +175,21 @@ WildMidiDecoder::WildMidiDecoder(const std::string file_name) {
 	config_file = "wildmidi.cfg";
 	found = FileFinder::Exists(config_file);
 
+	// wildmidi command line player default config
+	if(!found) {
```

Comment:
nit: space between if and parentheses
```suggestion
	if (!found) {
```

Reply:
thanks, applied.